switchmap/poller/snmp/mib/cisco/mib_ciscovtp.py

```
@@ -115,35 +119,47 @@
-        for key, value in values.items():
-            final[key]["vlanTrunkPortDynamicState"] = value
+        # Can Limit concurrent SNMP queries (can adjust according to need)
+        semaphore = asyncio.Semaphore(5)
+
+        async def limited_query(method, name):
+            """Rate limit SNMP query."""
+
+            async with semaphore:
+                try:
+                    return name, await method()
+                except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)